### DISCUSSION TITLE
Plastic materials categorization

### DISCUSSION THREAD

**Discussion by I-am-Erk** [CleverRaven/Cataclysm-DDA#47877] [Ideas]
I have long been thinking about how we could improve the currently over-vague material, "plastic", which encompasses everything from natural latex or milk casein to PVC or epoxy. This category is currently so vague that you could make a good argument for, say, human hair fitting in it. This makes any kind of detailed plastic crafting rules almost impossible to write.

For a long time I thought recycling categories might be a good way to split plastics, but today it occurred to me that we could instead split along two axes.
1. Is it a plastic we can melt and reform safely (thermoplastic), or not (thermoset plastic, I will call this "set plastic" from here on because the real term is very confusing).
2. Elasticity/hardness. I suggest splitting this into three loose categories, "rubber" for elastomers and stretchy materials, "plastic" for moderately soft and bendy solid plastics, and "resin" for fairly rigid and durable solid plastics.

These aren't perfectly representative axes, but they allow us to create something like six materials representing a few different types of plastics that are commonly used, giving us a lot more granularity in maybe a more meaningful way. 

**set rubber**: eg natural latex. Tires would be this material for example.
**thermo rubber**: thermoplastic elastomers are commonly found in very soft plastics eg. the soft part of earbuds.
**set plastic**: this would make up most common plastic casings, eg the keys and housing of the computer I'm typing on right now. Bakelite would fit.
**thermo plastic**: the most common recyclable plastics eg. ABS, polystyrene and PET would fall in here. Also a lot of what's used for 3d printing.
**set resin**: for example epoxy, formica, duroplast: a lot of the strong, non-meltable materials used in cars and countertops and such. I think casein plastic fits here too.
**thermo resin**: the tougher meltable plastics like PVC and polycarbonate. Also the rest of the 3d printing plastics.

There's some argument that we could have more than two axes here, but I kinda think this covers most common useage examples; there's maybe some room for brittle not-strong plastics but I'm not sure that adds much. With a system like this we could rationally work out some plastic salvaging and reforming: for example, you can melt and reform PVC with the right knowledge and tools, but you can't bash a laboratory countertop and get 20 fragments of plastic that can be made into car tires. We could have some chemistry recipes for reasonable plastic types you might be able to produce, too. The nice thing is that we don't need to have big discussions over whether object X is made of Y specific plastic type, we can base it mostly on the feel of the plastic and the basic estimate of how tough or soft it is. The only trick is figuring out what can be melted and what can't.

Anyone with more knowledge of plastic materials want to pitch in thoughts?

**Comment by chrispikula** [comment]
So just for information, quoting myself off the discord after a wiki walk:

Plastic numbers 1 & 2 (the easily _recyclable_ kinds), make up 34 & 18% of plastic production, however 2/3s of that later is for fibers, like buried cables.  Number 2, HDPE, is what we'd consider thermo-plastic, what we could conceivable heat-reform.  4 is also situationally reusable, but I'll talk about that in a moment.

So you're looking around 6% of plastics that you'd recover actually be easily reformable.  A good chunk of the type-1 would be reusable, in terms of making insulation or wall filler material, but requires specialized equipment (high temp pressure vessels), to reform into specific shapes.

Number 4, LDPE, which is basically shopping bags and cling wrap, can be used for thermo forming, but it can't be used on it's own.  Couple pieces of construction paper and a clothes iron and you can make a more rigid sheet.  Or you can use an organic fiber towel, shape it, then use the paper&iron to kinda impregnate the cloth, making a stiff, rigid structure.  Say, a water resistant backpack.  It's kinda fragile, though, and will fracture with daily use.  I wouldn't use it for anything I'd consider mission-critical or even mission-nice to have.

The other types are pretty hard to work with.  We are talking Oil filters and phone cases.  Not worth the time, or effort, even in the apocalypse.

Also, we don't need to get rid of plastic chunks entirely.  Given that even recyclable plastic has a limited number of uses, we could emulate that with needing to resort anything that gets scrapped.  Sure it's all HDPE, but maybe some of it's no good anymore?  Too brittle, too stiff.

Summary:
We could have a 'sort' recipe for groups of plastic chunks, making filler and thermoplastic chunks  (Aka, numbers 2 and 4).  Or any other additional types/axis.  Then any new or updated recipes would need to use the new categories, and the old stuff would get obsoleted.  Useless types could be burned or used for aggregate, I guess?

Non-wikipedia information:
https://wedocs.unep.org/handle/20.500.11822/26745?show=full
Link for mapping_plastics.pdf is on that page.  Page 31 has a good chart of Production %'s.  Page 61 has a good table on 'what is used in what product.'  Page 95 has a good table on what is used in what industry.
Warning:  It's an awful 51Mb pdf, and is slow as hell in foxit when you scroll to chapter headings with huge background splashes.

Finding production percentages of types of plastic was a gigantic pain in the butt.  Nor do the numbers match up well with wikipedia.  /me shrugs.

**Comment by Fosheze** [comment]
This is only tangentially related but I figured I would make a note of it here for when someone starts changing recipes around for this.

Because tires would be "set rubber" which isn't reworkable for repairing said tires, then some sort of different method of repairing tires on vehicles should be available so we don't leave survivors stranded. Normal patch kits are basically just a small hunk of tarred rope you shove in the hole so you could probably make one out of "thermo rubber" and a short string using some sort of heating tool. Alternatively if the survivor was desperate and willing to go full redneck you can patch a hole in a tire with a coarse thread screw so in game probably just a piece of scrap metal (because we don't have screws in game) and a tool with screw driving.

**Reply by actual-nh** [reply]
IIRC, patching a bicycle tire essentially involves roughening the surface,  putting glue on, and then putting a patch on it. Bicycle tires (at least, for road bikes; not sure for mountain) are actually inflated to a _higher_ pressure than car tires, so that should work - might be bumpy, though, and heat resistance would be a definite worry.

**Reply by Fosheze** [reply]
> IIRC, patching a bicycle tire essentially involves roughening the surface, putting glue on, and then putting a patch on it. Bicycle tires (at least, for road bikes; not sure for mountain) are actually inflated to a _higher_ pressure than car tires, so that should work - might be bumpy, though, and heat resistance would be a definite worry.

Normally car tires have a much harsher tread so it would be harder to get a glued on patch to seal from the outside. I believe that is the reason that car emergency patch kits don't use the same method. Not to mention, car tires normally see quite a bit more wear than bike tires so the patch needs to be pretty thick if it's applied to the outside of the tire and it needs to be adhered better due to the higher speed. You should be able to use a flat glued on patch from the inside if you pull the tire off the rim though.

The only other possible issue that I could possibly see with a glued on patch on car tires is with how much the tires flex. Because bike tires are inflated to a higher pressure they don't flex as much while moving (citation needed). Due to that whatever adhesive used if used on a car tire needs to be able to repeatedly and rapidly flex without wearing out. Of course I'm not a professional tire patcher so don't quote me on that last part. I'm just speaking from personal experience of trying to redneck repair junk cars enough to get them home.

**Reply by actual-nh** [reply]
Oops. I was thinking of tires with tubes - the patch goes on the tube (although putting it on the inside of a tire should also work in many cases). Could still work with non-bicycles in some cases - motorcycles apparently sometimes have tubes inside of tires, even tires meant to be used without tubes.

**Reply by I-am-Erk** [reply]
The material type wouldn't have a huge impact on whether or not it was repairable. This is a feature that impacts recipes and stuff. Plenty of things (almost all things?) made of various thermoset plastics can be repaired: bicycle patch kits, yes... you can fill cracks in epoxy with more epoxy, you can superglue two pieces of bakelite together, etc. You just can't melt the plastic down and reform it into something new.
